tests/jballerina-unit-test/src/test/resources/test-src/object/object_function_pointer.bal

```diff
@@ -41,9 +43,31 @@ public class SampleWindow {
     }
 }
 
-function testObjectFunctionPointer() returns int {
+function testObjectFunctionPointer() {
     Window win = new LengthWindow();
     (function () returns int) pointer = () => win.process();
     win = new SampleWindow();
-    return pointer();
+    test:assertTrue(pointer() is int);
```

Comment:
No need for this assertion because `pointer` is a function which returns an int anyway.